requirements.txt

```diff
@@ -2,7 +2,7 @@ spacy>=3.0.0,<3.3.0
 matplotlib>=3.5.0,<4.0.0
 
 # large danish spacy pipeline
-https://huggingface.co/spacy/da_core_news_lg/resolve/main/da_core_news_lg-any-py3-none-any.whl
+https://huggingface.co/spacy/da_core_news_lg/resolve/main/da_core_news_lg-any-py3-none-any.whl # A very large download if you're not on a fast connection, maybe not a default requirement?
```

Comment:
It is not installed with the package, it is intended for the pytests. I could test it using a smaller model, but the smaller models are much more inconsistent when spaCy updated them, this will lead to fewer unintended errors.

Another problem would be to remove the parsing from the test, but I want to make sure that it works with the spaCy pipelines and not just a preparsed doc.

Reply:
```suggestion
https://huggingface.co/spacy/da_core_news_lg/resolve/main/da_core_news_lg-any-py3-none-any.whl
```